BrandonBeltran/hackerrank/03/JavaArrayList.java

```diff
@@ -0,0 +1,40 @@
+package bootcamp;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class JavaArrayList {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int collections = scanner.nextInt();
+        List <List> numbers = new ArrayList<>();
```

Comment:
The best practice in Collections or Map variable name, is use name in singular and them the interface name: numberList, change it in all the variables